lib/fastlane/plugin/release_actions/actions/set_constant_value.rb

```diff
@@ -0,0 +1,69 @@
+module Fastlane
+  module Actions
+    class SetConstantValueAction < Action
+      def self.run(params)
+        file = params[:file]
+        constant = params[:constant]
+        value = params[:value]
+        
+        file_contents = File.read(file)
+
+        CONSTANT_VERSION = /(?<=#{constant})(\s*=\s*["'].*)/
+  
+        if !file_contents.match(CONSTANT_VERSION)
+          UI.error("#{constant} not present or doesn't have an explicit value in #{file}")
+          return
+        end
+
+        new_value = " = '#{value}'"
+        
+        file_contents = file_contents.gsub(CONSTANT_VERSION, new_value)
+
+        File.open(file, "w") { |file| file.puts file_contents }
+        UI.success("Successfully modified #{constant} to value #{value} in #{file}")
+      end
+
+      #####################################################
+      # @!group Documentation
```

Comment:
Vat iz dis? For VSCode or something?

Reply:
Is "Documentation" the right term? I think Fastlane uses these internally to make decisions, too.

Reply:
> Vat iz dis? For VSCode or something?

Haha I'm not sure. I'll get rid of it though